portfolio/src/main/webapp/index.html

```diff
@@ -5,15 +5,54 @@
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/13dae45c68.js" crossorigin="anonymous"></script>
   </head>
+  <header>
+    <h2 id="heading">Kai's Portfolio</h2>
```

Comment:
Why is this not inside of `<body>`?  `<header>` is "flow" content per the spec and belongs in a `<body>` element. Yes, browsers do their best with non-conforming content but they work better with correct content. Consider using a tool like [HTML Tidy](https://www.html-tidy.org) to validate HTML.